### ISSUE TITLE
Remove meetup links

### ISSUE THREAD

**Issue by jvanz** [HackerspaceBlumenau/hackerspaceblumenau.github.io#2]
For some reasons that I did not remember, our meetup group is dead. Thus, we can remove the link from our website. 

For keep track my meeting with possibles partners I've created a Google agenda. I think can be a good ideia allow other people to see this agenda. For that, this is the html to add in the site:

```
<iframe src="https://calendar.google.com/calendar/embed?src=hr5il8bai81ckj1etfj7l2mb10%40group.calendar.google.com&ctz=America/Sao_Paulo" style="border: 0" width="800" height="600" frameborder="0" scrolling="no"></iframe>
```

**Comment by ricardobrg** [comment]
Fixed by e45371864219ec96decacc0dfdd2cc87942fc45e

**Comment by jvanz** [comment]
Take a look in the agenda page: http://hackerspaceblumenau.github.io/agenda/

The title is being shown twice
